src/utils/element-utils.ts

```diff
@@ -1,8 +1,8 @@
 import { TOOLBAR_CONTAINER_CLASS, TOOLBAR_ID } from '../constants'
 
-export function isElementInToolbar(el: Element): boolean {
+export function isElementInToolbar(el: Element | null): boolean {
     // NOTE: .closest is not supported in IE11 hence the operator check
```

Comment:
totally fly-by note for myself that this comment is probably out of date now that we're making ES6 and ES5 bundles separately